hooks/boot/20-certs_update.rb

```diff
@@ -18,6 +18,14 @@
   :default => false
 )
 app_option(
+  '--certs-reset',
+  :flag,
+  "This option will reset any custom certificates and use the self-signed CA " \
+  "instead. Note that any clients will need to be updated with the latest " \
+  "katello-ca-consumer RPM.",
```

Comment:
I already added it to the --certs-reset help, and I'll update the documentation on the web site